.gitignore

```
@@ -91,3 +91,5 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+.requirements.txt.swp
```

Add a blank line at the end of the file. 
```suggestion
.requirements.txt.swp

```
It's advisable to have files generated by your system such as `swp files` ignored inside your `global git ignore file`
here is a resource on how to add a global gitignore file. 
https://help.github.com/en/articles/ignoring-files